src/PubNub/PubNub.php

```diff
@@ -478,9 +480,9 @@ public function getLogger()
     }
 
     /**
-     * @param Logger $logger
+     * @param LoggerInterface $logger
      */
-    public function setLogger($logger)
+    public function setLogger(LoggerInterface $logger)
```

Comment:
To be compatible with `psr/log` `^3.0`:
```suggestion
    public function setLogger(LoggerInterface $logger): void
```